--- proj.copyright	2004-08-13 10:28:21.000000000 -0700
+++ proj.copyright.fedora	2004-08-13 10:39:08.000000000 -0700
@@ -1,3 +1,15 @@
+This package was created for Fedora by David M. Kaplan dmkaplan@ucdavis.edu on
+Fri Aug 13 10:31:34 PDT 2004.
+
+It was downloaded from ftp://ftp.remotesensing.org/pub/proj/proj-4.4.8.tar.gz
+
+Below is the version of the copyright that appeared with the Debian
+version of this package.  It explains the complete history of this
+package, including the transfer of the license from a governmental
+"public domain" license to an MIT-style license.
+
+==================Original Debian copyright file below======================  
+
 This package was debianized by Peter S Galbraith psg@debian.org on
 Sun, 24 Oct 1999 14:13:26 -0400.
 
